contents/docs/privacy/ad-blockers.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Tracking endpoints and ad blockers
+sidebarTitle: Ad blockers
+sidebar: Docs
+showTitle: true
+---
+
+PostHog provides a range of services that help teams build great products.
+
+Part of that range are web and product analytics, which send usage event reporting back to PostHog. We encourage our customers to be transparent about this reporting, provide several [privacy-preserving options](/docs/product-analytics/privacy) in our tools, and fully comply with data privacy laws, including GDPR.
+
+Nevertheless, privacy-focused users of the web may wish to prevent any such data collection. Ad and tracking blockers should target these endpoints:
+
+`https://us.i.posthog.com/capture`
+`https://eu.i.posthog.com/capture`
+
+PostHog provides other tools unrelated to event reporting and analytics, and these can be vital to workings of the sites which rely on them. You **should not block** these endpoints, at risk of breaking site functionality:
+
+`https://us.i.posthog.com/decide`
```

Comment:
What about `https://us-assets.i.posthog.com/static` ?

Reply:
Oh, I didn't turn that one up! And an EU equivalent I'm guessing?

Reply:
yup, surfaced here: https://posthog.com/docs/advanced/proxy#reverse-proxy-requirements